### ISSUE TITLE
transfer learning strategy 

### ISSUE THREAD

**Issue by divastar** [HonglinChu/SiamTrackers#26]
Hi.
Can you please share some transfer learning strategy to start with
without training from scratch?
thank you

**Comment by HonglinChu** [comment]
transfer learning stratgegy？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HonglinChu/SiamTrackers"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年12月16日(星期三) 晚上6:26
收件人:&nbsp;"HonglinChu/SiamTrackers"<SiamTrackers@noreply.github.com&gt;;
抄送:&nbsp;"Subscribed"<subscribed@noreply.github.com&gt;;
主题:&nbsp;[HonglinChu/SiamTrackers] transfer learning strategy  (#26)





 
Hi.
 Can you please share some transfer learning strategy to start with
 without training from scratch?
 thank you

**Comment by divastar** [comment]
Hi linlin .
Thank you for responding.
Can I ask you a question about Siamese++?
So what I am actually doing in the training phase is to learn to categorize
for foreground/background
And the output would be the correlation map?
After combining the results from rpn blocks?
Thank you
Korrine

**Comment by divastar** [comment]
or actually they do  correlation on the proposal maps first channel by
channel and then learn the regression + classification loss for
forground/background?
thank you
Korrine

**Comment by divastar** [comment]
Hi Linlin.
another question:
In the paper they say that the learning rate for the Resnet is 10 times
smaller then for the RPN
So I wonder how they achieve that?
Is there another loss for the resnet part?
because on the other hand they say that they do the end to end training?
So how there is a second learning rate?
thank you
Korrine
